containers/sprm-to-anndata/context/main.py

```diff
@@ -0,0 +1,93 @@
+import argparse
+from glob import glob
+from pathlib import Path
+from os import makedirs
+import json
+from ast import literal_eval
+
+import anndata
+import zarr
+import pandas as pd
+import numpy as np
+from shapely.geometry import Polygon
+from ast import literal_eval
```

Comment:
duplicate import. Do you have linting in place? Maybe that's not something it catches...